index.html

```diff
@@ -123,7 +132,21 @@ <h4>Basic</h4>
 })
 
 $('#btnStopSimple').click(function() {
-  topbar.hide()  
+  topbar.hide()
+})</pre>
+          </section>
+          <h4>Delayed show</h4>
+          <section class="code">
+            <p>You can use <code>topbar.delayedShow(microseconds)</code> to delay the progress bar. If the topbar is hidden within the time in microseconds, it does not show up at all. This can make the user experience feel smoother:</p>
```

Comment:
```suggestion
            <p>You can use <code>topbar.delayedShow(milliseconds)</code> to delay the progress bar. If the topbar is hidden within the time in milliseconds, it does not show up at all. This can make the user experience feel smoother:</p>
```

Should this say milliseconds?

https://developer.mozilla.org/en-US/docs/Web/API/setTimeout